Landing page improvements (#100)
espressif/developer-portal: content/_index.md

    showViews : false
---

> "Welcome to the [Espressif Systems](https://espressif.com/) Developer Portal—your official hub for all things ESP32 and more. Explore our extensive collection of articles, workshops, and tutorials to enhance your development journey with the latest tools and insights."

Reviewer: ```suggestion
"Welcome to the [Espressif Systems](https://espressif.com/) Developer Portal—your official hub for all good things, such as the ESP32 and more. Explore our extensive collection of articles, workshops, and tutorials to enhance your development journey with the latest tools and insights."
```

> ## Featured Article

Reviewer: Somehow the singular form here doesn't look right. Suggest:

```suggestion
## Featured Articles
```

How about using the plural forms in the headings below as well, such as _Workshop Highlights_, etc?

> ## Featured Article

Reviewer: How about adding an introduction for the articles as well? Maybe something like:

Make sure to browse through our articles. To simplify navigation, use tags to find what is relevant for you.

Author: Thank you, @f-hollow for your review.
Yes, we can mention it.

{{< article link="/blog/meet-espressif-developer-portal/" >}}

<br>
{{< button href="#button" target="_self" href=/blog/ >}}
More Articles
{{< /button >}}

## Workshop Highlight

Dive into our latest workshops and master the skills you need to maximize the power of the ESP32.

{{< article link="/workshops/iot-introduction-with-rainmaker/" >}}

<br>
{{< button href="#button" target="_self" href=/workshops/ >}}
More Workshops
{{< /button >}}

> ## New Chip Support Status for ESP-IDF

Reviewer: We moved this support status page from ESP-IDF's GitHub because the status also mentions other projects, not just ESP-IDF. SUggest:

```suggestion
## New Chip Support Status
```

> Here you can follow the current status of the new chip support for the ESP-IDF.

Reviewer: How about something like this:

```suggestion
Here you can follow the current status of the new chip support in various Espressif SDKs and frameworks.
```
